.toys/release/.lib/release_requester.rb

```diff
@@ -345,7 +345,12 @@ def determine_commit_info
       @release_commit_title = "release: Release #{info.gem_name} #{info.new_version}"
       @release_branch_name = @utils.release_branch_name(info.gem_name)
     else
-      @release_commit_title = "release: Release #{@gem_info_list.size} gems"
+      @release_commit_title = "release: Release #{@gem_info_list.size} gems\n\n"
+      @gem_info_list.each do |info|
+        gem_line = " *  **#{info.gem_name} #{info.new_version}**"
+        gem_line += info.last_version ? " (was #{info.last_version})" : " (initial release)"
+        @release_commit_title += gem_line + "\n"
```

Comment:
The commit title is used for the pull request title as well as the commit message, so we probably can't make it multiple lines. However, I think I see where you're going with this; let me open an alternate PR.